hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/package.json

```
@@ -164,4 +167,4 @@
     ]
   },
   "proxy": "http://localhost:9888"
-}
+}
```

Restore the extra line at the end

```suggestion
}

```